packages/preset-create-react-app/.storybook/config.js

```diff
@@ -0,0 +1,8 @@
+import { configure } from '@storybook/react';
+
+const req = require.context('./test-files', true, /.story.(j|t)sx?$/);
```

Comment:
Let's use `.stories.*` to be consistent?

Reply:
Sure, I think I copied this from another preset - but I agree. Will do.